pkg/chartutil/create.go

```diff
@@ -96,6 +104,40 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+volumes: []
+  # - name: cache-volume
+  #   emptyDir: {}
+
+volumeMounts: []
+  # - name: cache-volume
+  #   mountPath: /cache
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 10001
+
+env: []
+  # - name: FOO
+  #   value: bar
+  # - name: BAZ
+  #   value: qux
+
+readinessProbe:
+  httpGet:
+    path: /
+    port: http
+livenessProbe:
```

Comment:
Same comment here. I don't think a livenessProbe should be enabled by default. Not all applications respond to HTTP requests (in fact, Tiller's one such example), and not all of them respond with a 200 OK on the root URL.

Reply:
true, just following what comes with helm create atm.
I can tweak this (and readiness) to be disabled by default. 

Should these also be made configurable to be something else than an `httpGet` ?

Reply:
I was suggesting that this should all be removed completely. Did you read my earlier comment?

Reply:
I did, I did. I'm sorry I don't see where you said to remove the probes completely.